RELEASE.md

```diff
@@ -0,0 +1,4 @@
+Release type: patch
+
+Add description of scalar types JSON, Base16, Base32, Base64.
+All scalar types are documented after this release.
```

Comment:
```suggestion
This releases updates the JSON scalar definition to have the updated `specifiedBy` URL.

The release is marked as minor because it will probably change the generated schema if you're using the JSON scalar.
```